Week1/assignments/usmaila-abdoul/Staircase.js

```diff
@@ -0,0 +1,16 @@
+const staireCase = (n) => {
+	const Arr = [];
+  
+  for (let i =0; i <= n; i++) {
+  	Arr.push(' ');
+  }
+  
+  for (let i = 0; i<= n; i++) {
+  	Arr.shift();
+    Arr.push('#');
+    var arr = Arr.join('')
```

Comment:
@usmailaabdoul avoid using var. This is due to `hoisting` which can cause issues sometimes in JS.
Try to read more about it